rts/lra/pom.xml

```diff
@@ -138,13 +138,13 @@
     </build>
 
     <modules>
+        <module>lra-service-base</module>
```

Comment:
Does this relate to https://developer.jboss.org/message/990612?et=watches.email.thread#990612

Reply:
I did see the forum question about lra-service-base but wasn't sure. I did add this change to make sure that the lra common module is the first thing to be built (which it should be) but I am not sure if it is related (just that the forum post made me check the code).